### ISSUE TITLE
NTR: Visceral tissue biopsy: sphingomyelin high (Lab: pathology: biopsies & morphology)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#9061]
New term request Visceral tissue biopsy: sphingomyelin high (f446):   UMLS: 1291141

**Comment by pnrobinson** [comment]
@MickeySegal
Adding new term: 
- ID: HP:6000292
- label: Elevated tissue sphingomyelin concentration
- parents: [Abnormal muscle tissue metabolite concentration (HP:0034370)](https://hpo.jax.org/app/browse/term/HP:0034370)
- pmid: 
- def: Concentration of sphoingomyelin in tissues above the upper limit of normal.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/9061
- synonyms:
